app/src/main/java/com/work/restaurant/data/source/local/eat/EatLocalDataSourceImpl.kt

```diff
@@ -0,0 +1,65 @@
+package com.work.restaurant.data.source.local.eat
+
+import com.work.restaurant.network.room.database.EatDatabase
+import com.work.restaurant.network.room.entity.EatEntity
+import com.work.restaurant.util.AppExecutors
+
+class EatLocalDataSourceImpl(
+    private val appExecutors: AppExecutors,
+    private val eatDatabase: EatDatabase
+) : EatLocalDataSource {
+
+
+    override fun getDataOfTheDay(
+        date: String,
+        callback: EatLocalDataSourceCallback.GetDataOfTheDay
+    ) {
+
+    }
+
+
+    override fun deleteEat(date: String, callback: EatLocalDataSourceCallback.DeleteEatCallback) {
+
+    }
+
+
+    override fun addEat(
+        date: String,
+        time: String,
+        type: Int,
+        memo: String,
+        callback: EatLocalDataSourceCallback.AddEatCallback
+    ) {
+        appExecutors.diskIO.execute {
+
+            val eatEntity = EatEntity(null, date, time, type, memo)
```

Comment:
```suggestion
            val eatEntity = EatEntity(data = date, time = time, type = type, memo = memo)          
```
null 보다는 기본값 지정이 어떤가요

Reply:
c1162ef 수정하였습니다.